### PR TITLE
Contact dto entity

### PR DESCRIPTION
**Pull request by bablukondangkel**
This contains Contact Dto Entity Classes

### CODE REVIEW

File: Common/yaml/Contacts.yaml [hunk 1]
```diff
@@ -0,0 +1,374 @@
+openapi: 3.0.3
+info:
+  title: Smart Contact Mangaer OpenAPI 3.0
+  description: The Project was about creating smart contact manager.Which can be specifically used for business model.This is a web application which can be used as a simple pluging to any existing project.|-
```
Comment (babugarnaik) [inline Common/yaml/Contacts.yaml]: It should be "The project is*"
Correct the spell of plugin
Remove the pipe and hyphen symbol at last

File: Common/yaml/Contacts.yaml [hunk 2]
```diff
@@ -0,0 +1,374 @@
+openapi: 3.0.3
+info:
+  title: Smart Contact Mangaer OpenAPI 3.0
```
Comment (babugarnaik) [inline Common/yaml/Contacts.yaml]: Correct the spell of manager

File: Common/yaml/Contacts.yaml [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+    name: Smart Contact Manager
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.11
+externalDocs:
+  description: Find out more about Smart Contact Manager
+  url: http://swagger.io
+servers:
+  - url: http://localhost:8082
+tags:
+  - name: contacts
```
Comment (babugarnaik) [inline Common/yaml/Contacts.yaml]: Keep first letter of tag name in capital and singular

File: Common/yaml/Contacts.yaml [hunk 4]
```diff
@@ -0,0 +1,374 @@
...
+    externalDocs:
+      description: Find out more about our store
+      url: http://swagger.io
+  - name: user
+    description: Operations about user
+paths:
+  /v1/contacts:
+    post:
+      tags:
+        - contacts
```
Comment (babugarnaik) [inline Common/yaml/Contacts.yaml]: Change the tag here and other paths as per the previous comment

File: Common/yaml/Contacts.yaml [hunk 5]
```diff
@@ -0,0 +1,374 @@
...
+        '500':
+            description: Internal Server Error
+            content:
+              application/json:
+                schema:
+                  type: string
+                  example:
+                    message: "Something went Wrong"
+    get:
+
```
Comment (babugarnaik) [inline Common/yaml/Contacts.yaml:95]: remove the extra line

File: Common/yaml/Contacts.yaml [hunk 6]
```diff
@@ -0,0 +1,374 @@
...
+        - name: limit
+          in: query
+          description: limit of records in one page for pagination
+          required: false
+          schema:
+            type: integer
+            example: 3 
+        - name: name
+          in: query
+          description: Enter the Name
```
Comment (babugarnaik) [inline Common/yaml/Contacts.yaml]: Correct the description here and below params

File: Common/yaml/Contacts.yaml [hunk 7]
```diff
@@ -0,0 +1,374 @@
...
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/contactList'
+
```
Comment (babugarnaik) [inline Common/yaml/Contacts.yaml]: Remove extra line

File: Common/yaml/Contacts.yaml [hunk 8]
```diff
@@ -0,0 +1,374 @@
...
+            description: Internal Server Error
+            content:
+              application/json:
+                 schema:
+                   type: string
+                   example:
+                     message: Server Error        
+components:
+  schemas:
+    contactList:
```
Comment (babugarnaik) [inline Common/yaml/Contacts.yaml:327]: Make the schema names consistent. All should either start with capital letter or small letter.

File: smartcontactmanager/src/main/java/com/scm/smartcontactmanager/Service/ContactServiceImpl.java [hunk 9]
```diff
@@ -0,0 +1,5 @@
+package com.scm.smartcontactmanager.Service;
+
+public class ContactServiceImpl implements ConatctService{
```
Comment (babugarnaik) [inline smartcontactmanager/src/main/java/com/scm/smartcontactmanager/Service/ContactServiceImpl.java:6]: Service class should be annotated with @Service

File: smartcontactmanager/src/main/java/com/scm/smartcontactmanager/controller/contact/response/ContactResponseDto.java [hunk 10]
```diff
@@ -0,0 +1,5 @@
+package com.scm.smartcontactmanager.controller.contact.response;
+
+public class ContactResponseDto {
```
Comment (babugarnaik) [inline smartcontactmanager/src/main/java/com/scm/smartcontactmanager/controller/contact/response/ContactResponseDto.java:3]: Add a comment line to this class saying you would add changes later.

**Review by babugarnaik — CHANGES_REQUESTED** [review]

File: Common/yaml/Contacts.yaml (hunk 1 above)
Comment (bablukondangkel) [reply Common/yaml/Contacts.yaml]: i have fixed.please review again.

Comment (bablukondangkel) [reply Common/yaml/Contacts.yaml]: i have fixed.please review again.

File: Common/yaml/Contacts.yaml (hunk 2 above)
Comment (bablukondangkel) [reply Common/yaml/Contacts.yaml]: fixed please review

File: Common/yaml/Contacts.yaml (hunk 3 above)
Comment (bablukondangkel) [reply Common/yaml/Contacts.yaml]: fixed please review

File: Common/yaml/Contacts.yaml (hunk 4 above)
Comment (bablukondangkel) [reply Common/yaml/Contacts.yaml]: fixed please review

File: Common/yaml/Contacts.yaml (hunk 5 above)
Comment (bablukondangkel) [reply Common/yaml/Contacts.yaml:95]: fixed please review

File: Common/yaml/Contacts.yaml (hunk 6 above)
Comment (bablukondangkel) [reply Common/yaml/Contacts.yaml]: fixed please review

File: Common/yaml/Contacts.yaml (hunk 7 above)
Comment (bablukondangkel) [reply Common/yaml/Contacts.yaml]: fixed please review

File: Common/yaml/Contacts.yaml (hunk 8 above)
Comment (bablukondangkel) [reply Common/yaml/Contacts.yaml:327]: fixed please review

File: smartcontactmanager/src/main/java/com/scm/smartcontactmanager/Service/ContactServiceImpl.java (hunk 9 above)
Comment (bablukondangkel) [reply smartcontactmanager/src/main/java/com/scm/smartcontactmanager/Service/ContactServiceImpl.java:6]: fixed please review

File: smartcontactmanager/src/main/java/com/scm/smartcontactmanager/controller/contact/response/ContactResponseDto.java (hunk 10 above)
Comment (bablukondangkel) [reply smartcontactmanager/src/main/java/com/scm/smartcontactmanager/controller/contact/response/ContactResponseDto.java:3]: fixed please review

**Review by babugarnaik — APPROVED** [review]

**Pull request merged**